src/main/scala/com/redislabs/provider/redis/redisFunctions.scala

```diff
@@ -390,6 +402,28 @@ object RedisContext extends Serializable {
     conn.close()
   }
 
+  /**
+    * @param arr hashName/k/vs which should be saved in the target host
+    *            save all the k/vs to hashName(list type) to the target host
+    * @param ttl time to live
+    */
+  def setHash2(arr: Iterator[(String, (String, String))], ttl: Int, redisConfig: RedisConfig,
```

Comment:
Sorry if I missed this but why are we using #2 in the function name?

Reply:
@jessicastewart-udemy Godd Question. I predict someone asks me about this name. :)
Actually I thought about, toRedisHashWithHashName or toRedisHashRDDHasHashName.
like these things. 

 so @jessicastewart-udemy Could you recommend valid name for it?
I will change.

Reply:
toRedisNamedHASH ❓

Reply:
@jessicastewart-udemy How about toRedisBulkHash?

Reply:
What bulk means in that context is unclear to me...@simalince what do you think?

Reply:
@charsyam @jessicastewart-udemy I don't have a strong feeling about the wording..
I'm not sure what "Named" means in this context. 
"Bulk" in this context means we can set hashes in bulk which is kind of what this function is doing. The original function `setHash` takes one hash value as a parameter and sets field(s) for just this one hash. We want the new function to set field(s) for multiple hashes that are built into the RDD. So, words like "bulk", "batch", "multi", "multiple" could all make some kind of sense but I'm not sure what is the most self-explanatory one 🤔